admin/syslog-ng/files/syslog-ng.init

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh /etc/rc.common
 # Copyright (C) 2006-2016 OpenWrt.org
 
-START=20
+START=12
```

Comment:
imho neither 12 nor 20 make sense here. 41 would be appropriate, as that's the point where we are supposed to have mounted everything syslog-ng might be writing to

Reply:
`/var` is a link to `/tmp`, so that gets mounted early on.

We could always add triggers so that if the network configuration changes or the filesystem changes that it gets reloaded...